prow/config/external_plugins_config.yaml

```diff
@@ -1008,6 +1020,24 @@ ti-community-label:
       - 'question'
       - 'wontfix'
       - 'make-local-great-again'
+      - 'affects-4.0'
+      - 'affects-5.0'
+      - 'affects-5.1'
+      - 'affects-5.2'
+      - 'affects-5.3'
+      - 'affects-5.4'
+      - 'affects-6.0'
+      - 'affects-6.1'
+      - 'affects-6.2'
+      - 'may-affects-4.0'
+      - 'may-affects-5.0'
+      - 'may-affects-5.1'
+      - 'may-affects-5.2'
+      - 'may-affects-5.3'
+      - 'may-affects-5.4'
+      - 'may-affects-6.0'
+      - 'may-affects-6.1'
+      - 'may-affects-6.2'
```

Comment:
Is it added in labels.yaml also?
Please check and ensure all the labels.
@jebter

Reply:
Existing labels have been removed